src/csharp/Microsoft.Spark.E2ETest/IpcTests/ML/Feature/BucketizerTests.cs

```diff
@@ -0,0 +1,77 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Spark.ML.Feature;
+using Microsoft.Spark.Sql;
+using Xunit;
+
+namespace Microsoft.Spark.E2ETest.IpcTests.ML.Feature
+{
+    [Collection("Spark E2E Tests")]
+    public class BucketizerTests
+    {
+        private readonly SparkSession _spark;
+
+        public BucketizerTests(SparkFixture fixture)
+        {
+            _spark = fixture.Spark;
+        }
+
+        [Fact]
+        public void TestBucketizer()
+        {
+            Bucketizer bucketizer = new Bucketizer("uid")
+                .SetInputCol("input_col")
+                .SetOutputCol("output_col")
+                .SetHandleInvalid("skip")
+                .SetSplits(new[] {Double.MinValue, 0.0, 10.0, 50.0, Double.MaxValue});
+
+            Assert.Equal("skip",
+                bucketizer.GetHandleInvalid());
+
+            Assert.Equal("uid", bucketizer.Uid());
+
+            DataFrame input = _spark.Sql("SELECT ID as input_col from range(100)");
+
+            DataFrame output = bucketizer.Transform(input);
+            Assert.Contains(output.Schema().Fields, (f => f.Name == "output_col"));
+
+            Assert.IsType<string>(bucketizer.GetInputCol());
+            Assert.IsType<string>(bucketizer.GetOutputCol());
+            Assert.IsType<double[]>(bucketizer.GetSplits());
```

Comment:
Set at the top
```csharp
double[] expectedSplits = new[] {Double.MinValue, 0.0, 10.0, 50.0, Double.MaxValue};
```

Then
```suggestion
            Assert.Equal(expectedSplits, bucketizer.GetSplits());
```

Reply:
Same comment for MultipleColumns() tests